Makefile

```diff
@@ -1,5 +1,6 @@
 DATADIR=$(PWD)/ignored/data
 TESTDATADIR=$(PWD)/testdata
+TESTDATADIR_SEG_FAIL=$(PWD)/testdata-seg-fault
```

Comment:
Test som pridal do Makefile, kedze sa to musi testovat z najvyssej urovne (aby sa spustil `run.sh`)